### PR TITLE
feat: add role profiler

### PR DESCRIPTION
**Pull request by gamename**
##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
profile_roles

##### ANSIBLE VERSION
```
ansible 2.2.0.0
  config file = /Users/tennis/PycharmProjects/inf-shared/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```

##### SUMMARY
This is a simple callback plugin to track the total time spent in each role during the run of a playbook.

### CODE REVIEW

File: lib/ansible/plugins/callback/profile_roles.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    tn = time.time()
+    return filled('%s (%s)%s%s' %
+                  (time_current, time_elapsed, ' ' * 7, time_total_elapsed))
+
+
+class CallbackModule(CallbackBase):
+    """
+    This callback module provides profiling for ansible roles.
+    """
+    CALLBACK_VERSION = 1.0
```
Comment (bcoca) [inline lib/ansible/plugins/callback/profile_roles.py]: this is not a 1.0 callback, its using 2.0 API

Comment (gamename) [reply lib/ansible/plugins/callback/profile_roles.py]: Fixed

**Comment by gamename** [comment]
@bcoca FIxed

**Comment by gamename** [comment]
@bcoca Should I leave this open, or close it?  Been sitting out here for 6 months now.

**Comment by bcoca** [comment]
sorry .. this dropped from my list .. lost in shuffle.

**Comment by sivel** [comment]
Looks like this referenced imports that no longer exist.  And fails several pep8 checks.

Probably a good idea to retrigger CI on stale_ci issues before merging.

**Comment by gamename** [comment]
@sivel Thanks for catching the errors.  Could you point me to the CI report where they're recorded?  No sure where to look.  Then, I'll address them.

**Pull request merged**